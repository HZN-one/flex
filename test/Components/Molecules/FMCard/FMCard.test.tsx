/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";
import { FMCard } from "@Molecules";

import { render } from "@testing-library/react";

describe("FMCard", () => {
  it("should match snapshot", () => {
    const { container } = render(<FMCard testID="card-example" />);
    expect(container).toMatchSnapshot();
  });
});
