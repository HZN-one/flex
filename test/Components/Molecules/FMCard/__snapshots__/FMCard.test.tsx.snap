// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FMCard should match snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-bhp9pd-MuiPaper-root-MuiCard-root"
    data-testid="card-example"
  >
    <div
      class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
    />
  </div>
</div>
`;
